CogniteSdk/src/Resources/Alpha/Simulators.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2023 Cognite AS
+﻿﻿// Copyright 2023 Cognite AS
```

Comment:
should we just remove the year altogether from the copyright? https://daniel.haxx.se/blog/2023/01/08/copyright-without-years/